### PR TITLE
add base documentation for all operator+config resources

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@kalexand-rh @bmcelvee ptal

this is my attempt to straighten out/provide a base for documenting our various cluster config + operator config resources which a cluster admin would interact with in the course of configuring their cluster.

File: installing/install_config/customizations.adoc [hunk 1]
```diff
@@ -5,91 +5,215 @@ include::modules/common-attributes.adoc[]
...
+|Namespace
+
+|Alertmanager
+|monitoring.coreos.com
+|main
+|Controls the alertmanager deployment parameters
+|openshift-monitoring
+
+
+|Authentication
```
Comment (deads2k) [inline installing/install_config/customizations.adoc]: We do not wish to document this for cluster-admins.  This is not an area where they are allowed to alter configuration and status is reported back through our clusteroperator, so they do not need to view this resource for status either.

File: installing/install_config/customizations.adoc [hunk 2]
```diff
@@ -122,19 +246,19 @@ that are based on many of these CRDs to add more functionality to your
 |AuthenticationOperatorConfig
 |authentication.operator.openshift.io
```
Comment (deads2k) [inline installing/install_config/customizations.adoc]: looks like it has leaked here too.  @bergerhoffer please remove references to 

1. kubeapiserver.operator.openshift.io
2. kubecontrollermanager.operator.openshift.io
3. kubescheduler.operator.openshift.io
4. openshiftapiserver.operator.openshift.io
5. authentication.operator.openshift.io
6. serviceca.operator.openshift.io

These are not resources for cluster-admins.  This is not an area where they are allowed to alter configuration and status is reported back through our clusteroperator, so they do not need to view this resource for status either.

File: installing/install_config/customizations.adoc (hunk 1 above)
Comment (bparees) [reply installing/install_config/customizations.adoc]: will remove

File: installing/install_config/customizations.adoc (hunk 2 above)
Comment (bparees) [reply installing/install_config/customizations.adoc]: this is commented out and doesn't appear in the rendered doc.  not sure why it was put here in the first place but i left it as a reference of the "global set of stuff".

**Comment by bparees** [comment]
@deads2k updated w/ the outcomes from our discussion

File: installing/install_config/customizations.adoc [hunk 3]
```diff
@@ -5,91 +5,175 @@ include::modules/common-attributes.adoc[]
...
 You complete most of the cluster configuration and customization after you
-deploy your {product-title} cluster. By default, a number of
+deploy your {product-title} cluster. A number of
 _configuration resources_
 //and _Custom Resources_
 are available.
 
-You modify the configuration resources to define the major components of the
-cluster, such as the networking configuration and the identity provider.
+You modify the configuration resources to configure the major components of the
```
Comment (deads2k) [inline installing/install_config/customizations.adoc]: minor question: components or features? I think of them as features.

File: installing/install_config/customizations.adoc [hunk 4]
```diff
@@ -5,91 +5,175 @@ include::modules/common-attributes.adoc[]
...
 ////
 You can also deploy Custom Resources (CRs) that are based on a number of
 Custom Resource Definitions (CRDs) that are deployed on your cluster.
 ////
 
 [id="configuration-resources_{context}"]
-== Configuration resources
+== Cluster configuration resources
+
+All cluster configuration resources are globally scoped (not namespaced) and named `cluster`.
```
Comment (deads2k) [inline installing/install_config/customizations.adoc]: can you add a comment (not visible in rendered docs I hope), that say that if you find yourself writing a help page that requires coordinated changes on two resources, they should tag api-approvers and get some help rethinking their API?

**Comment by deads2k** [comment]
minor comments and question

/lgtm
/hold
holding for others to have a shot.

File: installing/install_config/customizations.adoc (hunk 3 above)
Comment (bparees) [reply installing/install_config/customizations.adoc]: i think of them as components (the bits of code implementing the feature) but you're probably right that to a user it's a feature.  i'll change it.

File: installing/install_config/customizations.adoc (hunk 4 above)
Comment (bparees) [reply installing/install_config/customizations.adoc]: i.....can......  but i would imagine by the time someone is writing that doc the ship has already sailed.

your comment makes me think this has already happened? :)

Comment (bparees) [reply installing/install_config/customizations.adoc]: comment added.

**Comment by adambkaplan** [comment]
Devex components look good. Allowing other teams to weigh in.

**Comment by bparees** [comment]
@bmcelvee @kalexand-rh i think this has gotten all the engineering attention it's going to, ptal/merge.

**Comment by bparees** [comment]
remade this PR against master: https://github.com/openshift/openshift-docs/pull/15905

will cherry-pick that back to 4.1 and 4.2.

**Pull request closed without merging**